GameOfThrones/Models/Castle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Data.Entity;
+
+namespace GameOfThrones.Models
+{
+    public class Castle
+    {
+        public int CastleId { get; set; }
+        public string Name { get; set; }
+    }
+
+    public class House
+    {
+        public int HouseId { get; set; }
+        public string Name { get; set; }
+        public string Region { get; set; }
+        public string Words { get; set; }
+        public Castle CastleId { get; set; }
```

Comment:
Ah, I think the link between tables has fallen into a similar trap as Sean did on his. The main isssue here is, given the way the model has been build, `CastleId` is interpreted in the database as `CastleId_CastleId` which is bad naming convention as its being automatically assigned by EF.

Its probably best for you to ask Sean about how he approached this for his app.